gds/src/test/scala/edu/gemini/aspen/gds/TemporaryTest.scala

```diff
@@ -0,0 +1,10 @@
+package edu.gemini.aspen.gds.osgi
+
+import org.junit.Assert._
+import org.junit.Test
+
+class TemporaryTest {
```

Comment:
???

Reply:
The build failed when there weren't any tests to run. This was the path of least resistance.